### ISSUE TITLE
Zero coverage SNVs and their relation to deletions

### ISSUE THREAD

**Issue by emucaki** [bioinform/neusomatic#84]
We have set up Neusomatic using the pre-built 'general purpose' training network models and believe we have it running properly. Most of the somatic mutations called can be confirmed visually through IGV.

However, we found that very small proportion of the called variants (~1%) have no rare allele at all (AO=0), despite passing QC. The variant is not seen through IGV, nor is it within any of the VCFs created before the Ensemble step. For each of these false SNVs, I've found a corresponding deletion (~20-40nt in length) called pre-ensemble with a starting coordinate 1nt upstream of the SNV (e.g. if the false variant was called chrX:123456C>A, a deletion would have been called at position chrX:123455). I've evaluated 7 examples, and in all 7 cases, this corresponding deletion was present.  

Is this a known bug, and if so, do you know how it could be avoided in the future? Thank you.

**Comment by msahraeian** [comment]
Hi @emucaki 
Thanks for your comment. Can you send me a small bam file (maybe by restricting that to 50bps around this location) so that I can check this issue. BTW, if you want to use the pre-trained model, I would recommend to use one of the SEQC trained models listed in the README.

**Comment by emucaki** [comment]
Thank you for the quick response. I will need to get permission to send the sample BAM file, so it may be a few days before I can do so. 

We used the "NeuSomatic_v0.1.0_ensemble_WEX_100purity.pth" model as we are working with exome data. Would you recommend that we use one of the SEQC WGS ones?

**Comment by msahraeian** [comment]
@emucaki Thanks. I think the SEQC models should work better even for the WES data. I would recommend to give it a try.

**Comment by emucaki** [comment]
The problem I described seems to have been an issue specific to the "NeuSomatic_v0.1.0_ensemble_WEX_100purity.pth" model. I've repeated the process with the "NeuSomatic_v0.1.4_ensemble_SEQC-WGS-Spike.pth" model with the same data and these "zero rare allele" SNVs have disappeared from the VCF. I see other signs that the variants being called are of better quality overall (e.g. fewer duplicate mutations in non-related samples). 

Thank you very much for your help.
